go/vt/sqlparser/parse_test.go

```diff
@@ -2296,6 +2296,42 @@ var (
 	}, {
 		input:  "create table unused_reserved_keywords (dense_rank bigint, lead VARCHAR(255), percent_rank decimal(3, 0), row TINYINT, rows CHAR(10), constraint PK_project PRIMARY KEY (dense_rank))",
 		output: "create table unused_reserved_keywords (\n\t`dense_rank` bigint,\n\t`lead` VARCHAR(255),\n\t`percent_rank` decimal(3,0),\n\t`row` TINYINT,\n\t`rows` CHAR(10),\n\tconstraint PK_project PRIMARY KEY (`dense_rank`)\n)",
+	}, {
+		input:  "SELECT JSON_PRETTY('{\"a\":\"10\",\"b\":\"15\",\"x\":\"25\"}')",
+		output: "select json_pretty('{\\\"a\\\":\\\"10\\\",\\\"b\\\":\\\"15\\\",\\\"x\\\":\\\"25\\\"}') from dual",
+	}, {
+		input:  "SELECT JSON_PRETTY(N'{\"a\":\"10\",\"b\":\"15\",\"x\":\"25\"}')",
+		output: "select json_pretty(N'{\\\"a\\\":\\\"10\\\",\\\"b\\\":\\\"15\\\",\\\"x\\\":\\\"25\\\"}') from dual",
```

Comment:
We should instead use strings with starting with back tics (`) so that they are easier to read.